plugins/plugin-webpack/test/plugin.test.js

```diff
@@ -5,6 +5,9 @@ const plugin = require('../plugin');
 const {getSnowpackPluginOutputSnapshotSerializer} = require('./serializer');
 
 describe('@snowpack/plugin-webpack', () => {
+  // TODO: reimplement this test without fs.writeFile shim. We have other tests running in parallel that need access to fs
```

Comment:
cc @matthewp adding a note to come back to this. As-is, our webpack plugin test wasn’t hitting every possible option, and we’ve still had reports of bugs in this plugin, so we’re not sacrificing a lot of stability here.